app/views/shared/_error_messages.html.erb

```diff
@@ -1,12 +1,16 @@
 <% if resource.errors.any? %>
-  <div id="error_explanation" class="alert alert-danger">
-    <h6><%= pluralize(resource.errors.count, "error") %> prohibited this <%= resource.model_name.human %> from being
-      saved:</h6>
-
-    <ul>
-      <% resource.errors.full_messages.each do |message| %>
-        <li><%= message %></li>
-      <% end %>
-    </ul>
-  </div>
+  <% full_error_messages = resource.errors.full_messages %>
+  <% if full_error_messages.join(' ').downcase.include? "password" %>
```

Comment:
Please refactor this to keep logic in the controller (or a controller helper) and out of the erb files where possible, or open a github issue to clean up after this PR so we remember to do it :)

Reply:
Moved the logic to the controller :)

Reply:
:)